src/check_update.c

```diff
@@ -37,6 +37,7 @@ static void print_help(const char *name)
 	printf("Help Options:\n");
 	printf("   -h, --help              Show help options\n");
 	printf("   -u, --url=[URL]         RFC-3986 encoded url for version string and content file downloads\n");
+	printf("   -v, --versionurl=[URL]  RFC-3986 encoded url for version string download\n");
```

Comment:
For check-update there isn't content it would ever use so the option doesn't make sense here I don't think.